README.md

```diff
@@ -353,9 +353,6 @@ dependencies {
 }
 ```
 
-### Class Documentation
-[Full class documentation for the runtime library is published automatically to JitPack](https://javadoc.jitpack.io/com/github/livefront/sealed-enum/sealedenum/latest/javadoc/sealedenum/index.html)
```

Comment:
So what's the alternative to this then?

Reply:
At the moment, nothing available online. I've tried to make the automatic Jitpack docs work, but they seem to be extremely finicky and it doesn't help this project is multi-module.

I'd like to try out automatically publishing to GitHub pages (a la https://docs.travis-ci.com/user/deployment/pages/), but need to discuss with @skirchmeier.

Reply:
OK